hack/prepare-e2e.sh

```diff
@@ -32,6 +32,37 @@ items:
     kind: ServiceAccount
     name: tiller
     namespace: kube-system
+### E2E node configuration ###
+# Apply sysctl configuration required by Elasticsearch
+# XXX A hack until CronJob daemonset is available.
+# See https://github.com/kubernetes/kubernetes/issues/36601
+- apiVersion: extensions/v1beta1
+  kind: DaemonSet
+  metadata:
+    name: navigator-e2e-node-config
+    namespace: kube-system
+  spec:
+    template:
+      metadata:
+        labels:
+          app: navigator-e2e-node-config
+      spec:
+        containers:
+        - name: apply-sysctl
+          image: busybox:latest
+          resources:
+            limits:
+              cpu: 10m
+              memory: 8Mi
+            requests:
+              cpu: 10m
+              memory: 8Mi
+          securityContext:
+            privileged: true
+          command:
+          - "/bin/sh"
+          - "-c"
+          - "sysctl -w vm.max_map_count=262144; sleep 2073600;"
```

Comment:
Would it be worth re-setting this more frequently, say, every 60s?

It's unlikely to ever be an issue, but if some other tool comes along and changes it, nodes may not start. Not a major blocker as this is obviously only used for our e2e tests, but if we want to let this serve as an example it may be best 😄

Reply:
I was actually looking for a way to sleep forever and discovered that  `sleep infintity` is a thing....but not on busy box.
Turns that that `sleep infinity` actually means `sleep 2073600` according to https://stackoverflow.com/a/41655546

That's why I chose it.

I propose I leave this for now and modify in a future branch?